browsers.config.js

```diff
@@ -8,14 +8,13 @@ const defaultBrowsers = ['chrome', 'firefox'];
 const availableBrowsers = ['chrome', 'firefox', 'opera'];
 
 // BrowserStack browsers
-const defaultBSBrowsers = ['chrome', 'firefox', 'edge'];
-const previousBSBrowser = ['chrome_previous', 'firefox_previous', 'edge_previous'];
-const supportedBSBrowsers = [...defaultBSBrowsers, ...previousBSBrowser];
+const defaultBSBrowsers = ['chrome', 'firefox', 'safari'];
+const previousBSBrowser = ['chrome_previous', 'firefox_previous', 'safari_previous'];
+const supportedBSBrowsers = [...defaultBSBrowsers, ...previousBSBrowser, 'edge', 'edge_previous'];
 const availableBSBrowsers = [
   'default', // default browsers alias
```

Comment:
What's the point of having 'default' and 'supported' alias when arrays are merge in `supportedBSBrowsers` anyway ?

Reply:
To use aliases in the `yml` files in short argument, e.g. `npm run test -- --browserstack supported`. you can also add/remove browser in `browsers.config.js` to the testing without touching the `yml`